src/kirby/components/list/list.component.scss

```diff
@@ -10,80 +11,39 @@ $shadow-mask: 0 -6px 0px 0px get-color('white');
   &:not(.has-sections) {
     box-shadow: $shadow;
   }
-}
-
-:host(.rounded) {
-  &:not(.has-sections) {
-    border-radius: $border-radius;
 
-    li:first-child {
-      border-top-left-radius: $border-radius;
-      border-top-right-radius: $border-radius;
-    }
-
-    .list-header {
-      border-top-left-radius: $border-radius;
-      border-top-right-radius: $border-radius;
-
-      & + ul > li:first-child {
-        border-radius: 0;
-      }
-    }
-
-    li:last-child {
-      border-bottom-left-radius: $border-radius;
-      border-bottom-right-radius: $border-radius;
-    }
+  ion-list {
+    contain: inherit;
   }
 
-  &.has-sections {
-
-    li.row {
-      box-shadow: $shadow-mask, $shadow;
-      
-    }
-
-    li.row.selectable:hover {
-      background-color: get-color('background-color');
-    }
-
-    li.section-header + li.row
-    {
-      border-top-left-radius: $border-radius;
-      border-top-right-radius: $border-radius;
-      box-shadow: $shadow;
-    }
-
-    li.row.last-in-section {
-      border-bottom-left-radius: $border-radius;
-      border-bottom-right-radius: $border-radius;
-    }
-  }
-}
-
-.list-header {
-  background-color: get-color('white');
-  border-bottom: 1px solid get-color('background-color');
-}
-
-ul {
-  list-style-type: none;
-  padding-left: 0;
-  margin: 0;
-  font-size: font-size('s');
-
-  li {
-    padding: size('m');
+  .list-ios,
```

Comment:
This class is not known to us - you are using inner implementation details from Ionic...